validator/coordinator/src/main/java/tech/pegasys/teku/validator/coordinator/performance/DefaultPerformanceTracker.java

```diff
@@ -94,11 +93,12 @@ public void onSlot(UInt64 slot) {
       return;
     }
 
-    if (slot.mod(Constants.SLOTS_PER_EPOCH).isGreaterThan(UInt64.ZERO)) {
+    if (slot.mod(specProvider.getGenesisSpecConstants().getSlotsPerEpoch())
+        .isGreaterThan(UInt64.ZERO)) {
```

Comment:
This won't be right if the number of slots per epoch changes in the future.

I suspect we want:
```suggestion
    if (!specProvider.computeStartSlotAtEpoch(specProvider.computeEpochAtSlot(slot).equals(slot)) {
```